packages/manager/modules/billing-components/src/components/subscription-tile/subscription-tile.html

```diff
@@ -20,7 +20,9 @@
                 <billing-status service="$ctrl.serviceInfos"></billing-status>
             </div>
         </oui-tile-description>
-        <oui-tile-actions data-ng-if="!$ctrl.isLoading">
+        <oui-tile-actions
+            data-ng-if="!$ctrl.isLoading && $ctrl.serverType !== 'node'"
+        >
```

Comment:
**nitpick:** This is a generic component. So add a flag to display/hide and based on the specific type of product.
```suggestion
        <oui-tile-actions
            data-ng-if="!$ctrl.isLoading && $ctrl.displayServiceActions !== 'node'"
        >
```